spec/lib/wakes/redirect_mapper_spec.rb

```diff
@@ -178,4 +178,66 @@
         .to have_wakes_graph(:canonical_location => '/target', :legacy_locations => ['/source'])
     end
   end
+
```

Comment:
@chrismaddocks it looks like your code here is really just testing the `has_wakes_graph` matcher. Notice how all the other tests in this file have lines like `described_class.new '/source', '/target'`. That's what will actually test the redirect mapper.